spyder/widgets/projects/explorer.py

```diff
@@ -261,13 +262,16 @@ def delete_project(self):
 # Tests
 #==============================================================================
 class ProjectExplorerTest(QWidget):
-    def __init__(self):
+    def __init__(self, directory=None):

```

Comment:
Please make `directory` an attribute of this class, i.e.

    self.directory = directory